src/test/java/ProfileTests.java

```diff
@@ -1,64 +1,27 @@
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
 import org.testng.Assert;
 import org.testng.annotations.Test;
-import java.util.UUID;
+import pages.LoginPage;
+import pages.ProfilePage;
+
+import static java.sql.DriverManager.getDriver;
 
 public class ProfileTests extends BaseTest {
+    LoginPage loginPage = new LoginPage();
+    ProfilePage profilePage = new ProfilePage();
 
     @Test
-    public void successfulLoginTest() {
-        login("demo@class.com", "te$t$tudent");
-        openProfile();
-        enterCurrentPassword("te$t$tudent");
-        enterCurrentEmail("demo@class.com");
-        String newName = generateRandomName();
+    public void changeProfileNameTest() throws InterruptedException {
+        loginPage.login("demo@class.com", "te$t$tudent");
+        profilePage.openProfile();
+        profilePage.enterCurrentPassword("te$t$tudent");
+        profilePage.enterCurrentEmail("demo@class.com");
+        String newName = profilePage.generateRandomName();
         System.out.println(newName);
-        enterNewName(newName);
-        saveProfile();
-        driver.navigate().refresh();
-        String name = getProfileName();
+        profilePage.enterNewName(newName);
+        profilePage.saveProfile();
+        basePage.getDriver().navigate().refresh();
+        Thread.sleep(1000);
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.**